modules/core/src/smithy4s/http/HttpEndpoint.scala

```diff
@@ -26,6 +26,9 @@ trait HttpEndpoint[I] {
 
   // Returns a path template as a list of segments, which can be constant strings or placeholders.
   def path: List[PathSegment]
+
+  // Returns a map of static query parameters that are found in the uri of Http hint, there cannot be multiple of the same key !.
```

Comment:
even though that's the case at the smithy level, I'd be more comfortable with modelling it in this  interface as a multimap, to future-proof it in case it ever changes.

Reply:
fair enough , Ill revert